cmd/deploy.go

```diff
@@ -75,6 +75,10 @@ func init() {
 }
 
 func deploy(cmd *cobra.Command, args []string) error {
+	apiCluster := flagCluster
```

Comment:
Må sjekkes ved delete også?

Reply:
Ja, jeg har nok oversett bruken av --cluster for ao ad delete.

Reply:
Legges til tilsvarende for ao ad delete og ao ad redeploy